docs/usage/management-api-walkthrough/02_policies.md

```diff
@@ -15,8 +39,188 @@ Content-Type: application/json
   "@context": {
     "odrl": "http://www.w3.org/ns/odrl/2/"
   },
-  "@type": "PolicyDefinitionRequestDto",
-  "@id": "<ID_SET_BY_CLIENT>",
+  "@type": "PolicyDefinitionRequest",
```

Comment:
actually, it works. it probably shouldn't so I'll correct the docs here.

Reply:
now it does because the `@vocab` has been added, but when I reviewed it wasn't there

Reply:
yea sorry, I nuked the history by force-pushing. It works without vocab as well.

Reply:
sorry for nitpicking, but if you remove the `@vocab` [it will not expand correctly](https://json-ld.org/playground/#startTab=tab-expanded&json-ld=%7B%22%40context%22%3A%7B%22odrl%22%3A%22http%3A%2F%2Fwww.w3.org%2Fns%2Fodrl%2F2%2F%22%7D%2C%22%40type%22%3A%22PolicyDefinitionRequest%22%2C%22%40id%22%3A%22%7B%7BPOLICY_ID%7D%7D%22%2C%22policy%22%3A%7B%22%40type%22%3A%22odrl%3ASet%22%2C%22odrl%3Apermission%22%3A%5B%7B%22odrl%3Aaction%22%3A%7B%22%40id%22%3A%22odrl%3Ause%22%7D%2C%22odrl%3Aconstraint%22%3A%5B%7B%22odrl%3AleftOperand%22%3A%7B%22%40value%22%3A%22BusinessPartnerNumber%22%7D%2C%22odrl%3Aoperator%22%3A%7B%22%40id%22%3A%22odrl%3Aeq%22%7D%2C%22odrl%3ArightOperand%22%3A%22%3CbpnNumber%3E%22%7D%5D%7D%5D%7D%7D)

anyway, now it's correct, we're good.